pr_parser.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python
+
+"""
+pr_parser.py: SCRIPTS that analyse pr infomation and write parsed pr infomation
+(especially the multi-relate-pr) to post-build proprerties for downstream job.
+"""
+__author__ = "Alan Wei"
+
+
+import argparse
+import sys
+import os
+import github
+
+from github import Github
+
+class PrParser(object):
+
+    def __init__(self):
+        """
+        __repo - RackHD/on-xxxx, The repo name of which repo triggers the build 
+        __target_branch - master etc., The target branch associated with pr
+        __merge_commit_sha - origin/pr/1/merge, The merge_commit_sha associated with the pr
```

Comment:
I don't understand what's the meaning of `origin/pr/1/merge`, could you please elaborate?